style.css

```diff
@@ -316,6 +293,181 @@
     border: 1px solid #344563;
   }
 
+  /* Detail Section Starts Here */
+  .no-scroll {
+    overflow: hidden;
+  }
+
+  .blur-modal {
+    filter: blur(3px);
+  }
+
+  .detail-card {
+    display: none;
+  }
+
+  .detail-card--show {
```

Comment:
- [ ] Kindly specified in the previous review, Kindly make sure your popup modal is centered from the top when it is opened.
You may consider fixing this by readjusting your `margin`. you may consider removing the auto and setting it with a specific value.
![image](https://github.com/sulaimonaa/Portfolio/assets/97100091/b727a665-1c4a-4e82-9742-0902a73a543f)